packages/legend-query-builder/src/stores/QueryBuilderResultState.ts

```diff
@@ -68,7 +80,10 @@ export class QueryBuilderResultState {
   executionResult?: ExecutionResult | undefined;
   executionDuration?: number | undefined;
   latestRunHashCode?: string | undefined;
+  selectedCells: CellData[];
```

Comment:
these are grid logic, we should put them together as a block, why do we mingle `queryRunPromise` in the middle?